### ISSUE TITLE
Save Failed. The value cannot be null. Name of the value: collection

### ISSUE THREAD

**Issue by Megalith9** [kaiheilos/Utilities#3]
When I try to edit a .uasset file then save, an error like that appears.
Is there a way to fix it?
![image](https://user-images.githubusercontent.com/81762361/134270073-b90ce7ee-64de-4a78-9ea2-424793efa529.png)

**Comment by TheTrueTenoz** [comment]
The warning that is cut off is telling you the file won't save correctly.  Make sure the engine version of the game is selected from the drop down in the top right of ae

**Comment by Megalith9** [comment]
> The warning that is cut off is telling you the file won't save correctly. Make sure the engine version of the game is selected from the drop down in the top right of ae

I know the version, it's 4.18. 
It's already selected
![image](https://user-images.githubusercontent.com/81762361/134451161-bc7f5daa-89d2-464a-8a08-6577855e524f.png)

**Comment by Megalith9** [comment]
> The warning that is cut off is telling you the file won't save correctly. Make sure the engine version of the game is selected from the drop down in the top right of ae

![image](https://user-images.githubusercontent.com/81762361/134451193-77817aff-ab9c-418d-9d42-889d64ec899b.png)

The engine version is the correct one. (4.18)

**Comment by kaiheilos** [comment]
Could you provide the file it's failing to save, or possibly the name of the game and file path? It's likely trying to read a data format that I haven't encountered yet and needs some edits

**Comment by Megalith9** [comment]
Of course, although those files are from the PS4 version of the game "Ace Combat 7"
The PC version of the files are encrypted and I haven't found a way to decrypt them.
[uassets.zip](https://github.com/kaiheilos/Utilities/files/7228750/uassets.zip)

**Comment by Megalith9** [comment]
And these are the PC uassets for you to check out.
[pcuassets.zip](https://github.com/kaiheilos/Utilities/files/7228756/pcuassets.zip)
